core/css.js

```diff
@@ -592,7 +592,8 @@ Blockly.Css.CONTENT = [
 
   '.blocklyWidgetDiv .goog-menuitem-content,',
   '.blocklyDropDownDiv .goog-menuitem-content {',
-    'font: normal 13px Arial, sans-serif;',
+    'font-size: 13px;',
+    'font-family: Arial, sans-serif;',
```

Comment:
Why break up property into font-size font-family?
If necessary, then minor nit (although rule is not followed in previous declarations) css style guide asks for declarations to be alphabetized (so swap family and size order).

Reply:
It is necessary so that it can be overridden by the renderer (via a more specific selector).
Noted on alphabetical order, will fix.